### ISSUE TITLE
[Bug Report more of a Question] Unsupported Video

### ISSUE THREAD

**Issue by Bagel94** [ChurchApps/FreeShow#911]
All our Motion Backgrounds are in the MP4 format, whenever a slide uses one, a little popup in the bottom right keeps saying "Unsupported Video!". Is MP4 not a supported format, even though it still plays the files?

**Version (Optional)**
 - OS: [Windows 10/11]
 - FreeShow Version: [1.3.0-beta.1]

**Comment by vassbo** [comment]
Yep, just ignore it. Had to remove the new Codec receiver before release and never updated that, so all videos are marked as "unsupported". Will be fixed!
